### DISCUSSION TITLE
貝多芬頭號遺作大功告成，AI 執筆譜《第十號交響曲》

### DISCUSSION THREAD

**Discussion by 239216** [1101-GS4719-C/Python-Community-GS4719-C#75] [General]
為了讓AI有跡可循，AI開始作曲前，團隊音樂專家整理貝多芬留下來的《第十號交響曲》筆記，大致拼湊貝多芬對《第十號交響曲》的想法，並設法將現存小節按樂章簡單分類。
同時AI專家也幫助AI「認識貝多芬」，從各種遺留作品與草稿學習貝多芬的創作脈絡，許多曾影響貝多芬的音樂前輩作品，也拿來當作幫助AI認識貝多芬的教材。除此之外，專家還得教會AI如何「作曲」，讓它熟悉交響曲基本曲式、將簡單旋律發展為完整作品的方法，以及如何搭配樂器等學問。
團隊替AI安排兩次「期中考」，分別由一位鋼琴家和一支弦樂四重奏樂團試演貝多芬原創，以及AI創作組成的新版《第十號交響曲》片段，讓台下坐著的記者、音樂學家和貝多芬專家嘗試分辨哪些部分是AI所做。
團隊一位音樂專家甚至形容，看著AI作曲，「就好像看一位孜孜不倦的學生每天不斷練習、學習，表現也越來越出色」。
https://technews.tw/2021/10/10/ai-completes-beethoven-symphony-no10/

**Comment by HSU-YU-JEN** [comment]
原來現在AI的程度已經可以學習作曲了~
